src/components/Header/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Header = () => {
+  return <div>NAVBAR HERE</div>;
+};
+
+export { Header };
```

Comment:
When you export the main component of a file, export it as default.

`export default Header`